helm/content-fluentd/templates/daemonset.yaml

```diff
@@ -26,6 +26,27 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         resources:
 {{ toYaml .Values.resources | indent 12 }}
+        env:
+        - name: KINESIS_STREAM
+          valueFrom:
+            configMapKeyRef:
+              name: global-config
+              key: content.fluentd.kinesis.stream
+        - name: AWS_ACCESS_KEY
```

Comment:
I don't think this is the right time to do this, but I'll mention it anyway for info purposes, since I see this is a new AWS user. If we'd like to get rid of them at some point, we can use https://github.com/jtblin/kube2iam and only use AWS roles. The advantage would be that we won't have any keys that can leak & that should be rotated.

Reply:
Interesting! I chatted with Euan about whether roles were an option (they're recommended in the fluentd plugin docs), and he said, possibly but not now. Definitely open to using this as a guinea pig if we ever want to experiment with kube2iam in the future?